src/current/v24.1/molt-fetch.md

```diff
@@ -379,14 +574,25 @@ molt fetch \
 - `--bucket-path` specifies a directory on an [Google Cloud Storage bucket](#data-path) where intermediate files will be written.
 - `--use-copy` specifies that `COPY FROM` is used to load the tables, keeping the source tables online and queryable but loading the data more slowly than `IMPORT INTO`.
 - `--cleanup` specifies that the intermediate files should be removed after the source data is loaded.
+- `--ongoing-replication` starts continuous [replication](#replication) of data from the source database to CockroachDB after the fetch process succeeds.
```

Comment:
Does it make sense to capture this into a variable or include for reuse? It's repeated verbatim.